conda_build/build.py

```diff
@@ -368,14 +368,13 @@ def build(m, get_src=True, verbose=True, post=None):
                 _check_call(cmd, env=env, cwd=src_dir)
 
     if post in [True, None]:
-        if post == True:
-            with open(join(config.croot, 'prefix_files.txt'), 'r') as f:
-                files1 = set(f.read().splitlines())
-
+        if exists(config.info_dir):
+            shutil.rmtree(config.info_dir)
+        with open(join(config.croot, 'prefix_files.txt'), 'r') as f:
+            files1 = set(f.read().splitlines())
```

Comment:
> Shouldn't this still be only if post == True?

That was the behaviour before, yes. In practice, it makes no difference, and would make sense for the post code to be as similar as possible to the actual code (for instance, you want to know if there is a bug with the writing of the ``prefix_files.txt`` file, and with this change, all builds will make use of it).